tests/libvmi/network.go

```diff
@@ -94,21 +97,20 @@ func InterfaceDeviceWithSRIOVBinding(name string) kvirtv1.Interface {
 	}
 }
 
-// InterfaceDeviceWithPasstBinding returns an Interface named "default" with passt binding.
-func InterfaceDeviceWithPasstBinding(ports ...kvirtv1.Port) kvirtv1.Interface {
+// InterfaceDeviceWithPasstBinding returns an Interface named "default" with passt binding plugin.
+func InterfaceDeviceWithPasstBindingPlugin(ports ...kvirtv1.Port) kvirtv1.Interface {
 	return kvirtv1.Interface{
-		Name: DefaultInterfaceName,
-		InterfaceBindingMethod: kvirtv1.InterfaceBindingMethod{
-			Passt: &kvirtv1.InterfacePasst{},
-		},
-		Ports: ports,
+		Name:    DefaultInterfaceName,
+		Binding: &kvirtv1.PluginBinding{Name: PasstBindingName},
+		Ports:   ports,
 	}
 }
 
-func InterfaceDeviceWithBindingPlugin(name string, binding kvirtv1.PluginBinding) kvirtv1.Interface {
+func InterfaceDeviceWithBindingPlugin(name string, binding kvirtv1.PluginBinding, ports ...kvirtv1.Port) kvirtv1.Interface {
```

Comment:
I did not detect who is using this new thing now. Is it a leftover?

Reply:
macvtap binding plugin test.